datasetinsights/evaluation_metrics/average_mean_square_error.py

```diff
@@ -0,0 +1,50 @@
+r"""Average Mean Square Error metrics.
+
+The Average mean square error can be described as:
+
+.. math:: // TODO fix
+    \{\frac{1}{n}\sum_{p}^{n}{(y_p-\hat{y_p})}^2}
+"""
+
+import numpy as np
+
+from .base import EvaluationMetric
+from .exceptions import NoSampleError
+
+
+class AverageMeanSquareError(EvaluationMetric):
```

Comment:
Can you add a unittest for this? Here is an [example](https://github.com/Unity-Technologies/datasetinsights/blob/master/tests/evaluation_metrics/test_average_relative_error.py).